classes/game.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'item'
+
+class Game < Item
+  def initialize(publish_date, multiplayer, last_played_at)
+    super(publish_date)
+    @multiplayer = multiplayer
+    @last_played_at = DateTime.parse(last_played_at)
+  end
+
+  def last_played_old
+    return true if @last_played_at < DateTime.now.next_year(-2)
+  end
+
+  private :last_played_old
+
+  def can_be_archived?
+    return true if last_played_old && super
+  end
```

Comment:
Great work with the code. I believe it's looking fine and it works as expected. 👍 

## Request

I suggest you to make both methods private since, according to the diagram, the can_be_archived?() method needs to be private in the Game class. (Currently it is possible to access it by calling it, for example doing `Game.new.can_be_archived?`

## Optional:

I believe it would be a great idea to make your code a little shorter by skipping the last_played_old method definition.

In the can_be_archived?, you could do:

`return true if super && @last_played_at < DateTime.now.next_year(-2)`

It believe it would look even better.